decidim-assemblies/app/views/layouts/decidim/admin/assembly.html.erb

```diff
@@ -13,7 +13,7 @@
           <ul id="components-list">
             <% current_participatory_space.components.each do |component| %>
               <% if component.manifest.admin_engine && user_role_config.component_is_accessible?(component.manifest_name) %>
-                <li <% if is_active_link?(manage_component_path(component)) || is_active_link?(decidim_admin_assemblies.edit_component_path(current_participatory_space, component)) || is_active_link?(decidim_admin_assemblies.edit_component_permissions_path(current_participatory_space, component)) %> class="is-active" <% end %>>
+                <li <% if is_active_link?(manage_component_path(component)) || is_active_link?(decidim_admin_participatory_processes.edit_component_path(current_participatory_space, component)) || is_active_link?(decidim_admin_participatory_processes.edit_component_permissions_path(current_participatory_space, component)) || participatory_space_active_link?(component) %> class="is-active" <% end %>>
```

Comment:
In general, looks good for me @slickepinne 
Some minor doubt here: why we use `decidim_admin_participatory_processes` to check paths instead of `decidim_admin_assemblies`?
I saw that this happens in:
- decidim-assemblies/app/views/layouts/decidim/admin/assembly.html.erb
- decidim-conferences/app/views/layouts/decidim/admin/conference.html.erb
- decidim-initiatives/app/views/layouts/decidim/admin/initiative.html.erb
The only place where I find that make sense is in file:
- decidim-participatory_processes/app/views/layouts/decidim/admin/participatory_process.html.erb
Thanks in advance!

Reply:
Thank you, @Leusev! Yeah, it wasn't meant to be this way. 😅 I updated it.